public/assets/press/starlight-logo-dark.svg

```diff
@@ -0,0 +1,35 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 200 55">
```

Comment:
Run these through [`svgomg`](https://jakearchibald.github.io/svgomg/)?

Reply:
I did 😅 

There are probably a few small wins — I don’t think SVGO is able to deduplicate identical gradient filters — but not much.